index.js

```diff
@@ -41,80 +41,58 @@ enabled. This way we can avoid paying for pre-processing if the value would not
 have been logged anyway.
 */
 
-const debug = require('debug')
+const stringify = require('safe-json-stringify')
 
 // This is the directory from which node process was launched.
 // We use it as root since it gives relative paths which can be clicked on
 // (in terminals such as iTerm) to open the file in an editor.
 const cwd = process.cwd()
 
-expandThunks = (array) => array.map(
+const expandThunks = (array) => array.map(
   elem => typeof elem == 'function' ? elem() : elem
 )
 
-makeLazyLogger = (strictLogger) => {
+const makeLazyLogger = (strictLogger) => {
   return strictLogger.enabled ?
     // if logger enabled, call it with functions expanded to actual values
     (...args) => strictLogger.apply(strictLogger, expandThunks(args)) :
     // no-op
     () => {}
 }
 
-makeStrictLogger = (label, {logToStderr, isErrorLogger}) => {
-  const logger = debug(label)
-  if (!logToStderr && !process.env.DEBUG_LOG_TO_STDERR) {
-    // by default log to stdout
-    logger.log = console.log.bind(console);
+const parseArgs = (args) => args
+  .map(arg => stringify(
+    arg instanceof Error ? {
+      name: arg.name,
+      message: arg.message,
+      data: arg.data,
+      stack: arg.stack
+    } : arg)
+  ).join(' ')
+
+const makeStrictLogger = (severity, context) => {
+  const logger = (...args) => {
+    const message = parseArgs(args)
+    const payload = { severity, timestamp: Date.now(), context, message }
+    console.log(stringify(payload))
   }
 
-  if (isErrorLogger) {
-    const errorLogger = (...args) => {
-      if (!logger.enabled) return
-
-      const last = args[ args.length - 1 ]
-
-      if (last instanceof Error) {
-        const argsWithErrorMessage = args.slice(0, -1)
-        argsWithErrorMessage.push(last.toString())
-        // Log error.message via the logger with label.
-        logger.apply(logger, argsWithErrorMessage)
-        // Log error object on its own, so that it can be parsed by GCP Error
-        // reporting.
-        console.error(last)
-      }
-      else {
-        logger(args)
-      }
-    }
-
-    errorLogger.enabled = logger.enabled
-    return errorLogger
-  }
-
-
   return logger
 }
 
 module.exports = function (filename, options = {}) {
   const {
-    prefix = 'dvf',
     root = cwd,
-    logToStderr = false
```

Comment:
do we no longer need to print anything to stderr?

Reply:
I dont think we need, if we dump json with correct fields and GCP parses that

Reply:
and I checked codebase where this package was required and there were no cases when options were passed